shinydashboard/lantern/server.R

```diff
@@ -284,25 +307,43 @@ function(input, output, session) { #nolint
     return(res)
   })
 
-  output$show_resource_checkboxes <- renderUI({
-    if (show_resource_checkbox()) {
+  #                                          #
+  # Display Resource and Operations Checkbox #
+  #                                          #
+
+  output$show_resource_operation_checkboxes <- renderUI({
+    if (show_resource_checkbox() && show_operation_checkbox()) {
       fluidPage(
         fluidRow(
           h2("FHIR Resource Types"),
-          p("By default, the list of resources below contains the supported resources across all endpoints and FHIR versions. Remove a resource from the list by clicking the 'x' in each box.
-            You may also change the FHIR Version or Developer filtering criteria to select the applicable supported resources from the default list.
-            Any selected resources at that point will be removed if no endpoints that pass the selected filtering criteria support the given resource.
-            Resources that are filtered out of the selected list will not re-appear in the list if you make other changes to the FHIR Version or Developer filtering criteria.
-            You must either (1) select a resource from the resources drop down to add it to the list, or (2) click the 'Select All Resources' button to add all resources that are supported by the endpoints passing the selected criteria.", style = "font-size:19px; margin-left:5px;"),
-          p("Note: This is the list of FHIR resource types reported by the CapabilityStatement / Conformance Resources from the endpoints. This reflects the most recent successful response only. Endpoints which are down, unreachable during the last query or have not returned a valid CapabilityStatement / Conformance Resource, are not included in this list.", style = "font-size:15px; margin-left:5px;"),
-          selectizeInput("resources", "Click in the box below to add or remove resources:", choices = checkbox_resources_no_filter(), selected = checkbox_resources_no_filter(), multiple = TRUE, options = list("plugins" = list("remove_button"), "create" = TRUE, "persist" = FALSE), width = "100%"),
-          actionButton("selectall", "Select All Resources", style = "margin-top: -15px; margin-bottom: 20px;"),
-          actionButton("removeall", "Remove All Resources", style = "margin-top: -15px; margin-bottom: 20px;")
+          column(width = 4,
+            multiInput(
+              inputId = "resources",
+              width = "500px",
+              label = "Click in the box below to add or remove resources:",
```

Comment:
I think we should change this (and on lines 358, 368, 375, and 384) to be "Click a resource on the left to add, and on the right to remove"